.gitignore

```diff
@@ -19,3 +19,8 @@ src/guides/install/auspice-install.md
 src/guides/install/cli-install.md
 src/tutorials/narratives-how-to-write.md
 src/tutorials/SARS-CoV-2/steps/*.md
+src/learn/nextclade.md
```

Comment:
This doc doesn't exist. Maybe it used to? See below for comments on how to potentially include the algorithm docs.
```suggestion
```